python/tvm/target/target.py

```diff
@@ -182,6 +182,10 @@ def check_and_update_host_consist(target, host=None, target_is_dict_key=True):
         target_is_dict_key : Bool
             When the type of target is dict, whether Target is the key (Otherwise the value)
         """
+        if target is None:
+            if host is not None:
+                warnings.warn("Target host is not empty when target is empty.")
```

Comment:
Should this be a warning or an assertion, if target=None and host!=None doesn't make any sense?

Reply:
Good point, should be an assertion. I fixed it.